streamer/src/test/utils/postgres/postgresQueries.js

```diff
@@ -0,0 +1,147 @@
+const {environment} = require('../../../environment')
+const DB_SCHEMA = environment.DB_SCHEMA
+const totalStakeThreshold = 1000000
+const totalRewardThreshold = 1000000
+const totalRewardPointsThreshold = 20
+
+
+async function getQuery(postgres, queryString) {
+    let queryResult
+    try {
+        queryResult = await postgres.query(queryString)
+    } catch (error) {
+        console.log(error.stack)
+        throw new Error(`Error executing query`)
+    }
+    return queryResult
+}
+
+async function getMissedBlocksCount(postgres) {
+    const missedBlocksCount = await getQuery(postgres, `SELECT max(id)-count(*) as missedBlocks FROM ${DB_SCHEMA}.blocks`)
+    return missedBlocksCount.rows[0].missedblocks
+}
+
+async function getTopBlock(postgres) {
+    const topBlock = await getQuery(postgres, `SELECT max(id) as topBlock FROM ${DB_SCHEMA}.blocks`)
+    return topBlock.rows[0].topblock
```

Comment:
rows[0] possibly undefined.

Reply:
The same as for https://github.com/p2p-org/polkadot-profit-transformer/pull/25#discussion_r622055305